storage/io/src/main/java/org/openscience/cdk/io/SDFWriter.java

```diff
@@ -67,29 +68,38 @@
     private Set<String>               propertiesToWrite;
 
     /**
-     * Constructs a new SDFWriter that writes to the given {@link Writer}.
+     * Create an SDfile writer that will output directly to the provided buffered writer.
      *
-     * @param   out  The {@link Writer} to write to
+     * @param wtr writer
      */
-    public SDFWriter(Writer out) {
-        if (out instanceof BufferedWriter) {
-            writer = (BufferedWriter) out;
-        } else {
-            writer = new BufferedWriter(out);
-        }
+    public SDFWriter(BufferedWriter wtr) {
+        this.writer = wtr;
         initIOSettings();
     }
 
     /**
-     * Constructs a new MDLWriter that can write to a given
-     * {@link OutputStream}.
+     * Create an SDfile writer, the provided writer is buffered
+     * if it's not an instance of BufferedWriter. For flush control
+     * etc please create with {@link BufferedWriter}.
+     *
+     * @param wtr writer
+     */
+    public SDFWriter(Writer wtr) {
+        this(ensureBuffered(wtr));
+        initIOSettings();
+    }
+
+    /**
+     * Create an SDfile writer, the provided output stream is wrapped
+     * in a UTF-8 buffered writer.
      *
-     * @param   output  The {@link OutputStream} to write to
+     * @param output out stream
      */
     public SDFWriter(OutputStream output) {
-        this(new OutputStreamWriter(output));
+        this(new OutputStreamWriter(output, StandardCharsets.UTF_8));
```

Comment:
Mmmm... why not go for ASCII or ISO-8859-1 or so? I know that's outdated, but this is the MDL formats we're talking about... not sure how many cheminf tool out there support UTF-8... just something to think about... I checked the 2005 ctfile.pdf and it does not seem to depend a certain char set...